clients/stellar-relay-lib/src/connection/connector/connector.rs

```diff
@@ -424,13 +419,4 @@ mod test {
 		connector.handshake_completed();
 		assert!(connector.is_handshake_created());
 	}
-
-	#[tokio::test]
-	#[serial]
-	async fn enable_flow_controller_works() {
```

Comment:
Not quite sure how to re-introduce this test, or equivalent. Since the `enabled` state will be set when on response of the `Auth` message, so we would have to start the message polling process.

Reply:
I guess we don't need it anymore.